### ISSUE TITLE
Setting the statusline color

### ISSUE THREAD

**Issue by Vosjedev** [akinomyoga/ble.sh#400]
I was searching for an option to set the backgroud color, or even better in my case, disable the background. I would suggest adding this as the `prompt_status_line_background` and disabling the backgroundcolor if set to empty.

**Comment by akinomyoga** [comment]
There is already a setting: see [face `prompt_status_line`](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A74-Editing#user-content-face-prompt_status_line).

```bash
# blerc

ble-face prompt_status_line=none
```

You can basically put the setting in `~/.blerc`. If you try that on the command line, to make it reflected, you need to reset bleopt `prompt_status_line` after changing face `prompt_status_line`. Maybe I can adjust ble.sh so that the change to face `prompt_status_line` in the command line is immediately reflected.

**Comment by Vosjedev** [comment]
Ah, I didn't know about that option existing, missed it while browsing the `.blerc` template. Sorry!

**Comment by akinomyoga** [comment]
Thanks.

> Maybe I can adjust ble.sh so that the change to face `prompt_status_line` in the command line is immediately reflected.

I have implemented this in commit f3e7e386cb1b9b5eb464e1bc23bb882b03f2d25d

> missed it while browsing the `.blerc` template

Also, in the same commit, I moved the face to the place where `bleopt prompt_status_line` is explained in `blerc.template`. Other faces are also moved to relevant places in `blerc.template`.

I also added a note in the item of [`bleopt prompt_status_line`](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A74-Editing#user-content-bleopt-prompt_status_line) in the wiki.
